tests/mainEndpoints.spec.ts

```
@@ -0,0 +1,79 @@
+  intercomUrl,
+  eventFeedUrl,
+] = apiObjsToTest.map((apiObj) => {
+  if (apiObj?.appId) appIds.push(apiObj?.appId);
+  return apiObj?.url;
+});
+
+const [intercomAppId] = appIds;
+
+test(`Check ${mainConfigUrl} availability`, async ({ request }) => {
+  expect(mainConfigUrl).toBeDefined();
+  const response = await request.get(mainConfigUrl!);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)